### PR TITLE
Use Livestream API queries on content, Following and Channel fragments

### PR DESCRIPTION
**Pull request by kekkyojin**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change

## PR Type

What kind of change does this PR introduce?

- [x] Feature
- [x] Refactoring (no functional changes)

## Fixes

Issue Number: #87 -partially- and #92 

## What is the current behavior?
Livestreams are extracted from claim_search queries
## What is the new behavior?
Livestreams are fetched using Odysee Livestream API. This allows to show scheduled livestreams and amount of viewers of the broadcast also.

### CODE REVIEW

**Review by akinwale — APPROVED** [review]
Looks good. Can merge after updating the `vh.getAbsoluteAdapterPosition()` calls.

File: app/src/main/java/com/odysee/app/adapter/ClaimListAdapter.java [hunk 1]
```diff
@@ -473,20 +492,20 @@ public void onBindViewHolder(ClaimListAdapter.ViewHolder vh, int position) {
         int type = getItemViewType(position);
 
         if (style == STYLE_SMALL_LIST) {
-            int paddingTop = position == 0 ? 16 : 8;
-            int paddingBottom = position == getItemCount() - 1 ? 16 : 8;
+            int paddingTop = vh.getAbsoluteAdapterPosition() == 0 ? 16 : 8;
```
[the remark this answers is not in the document]
Comment (kekkyojin) [reply app/src/main/java/com/odysee/app/adapter/ClaimListAdapter.java:495]: That’s what I thought.

IDE was showing a linting error -not a warning- on the ‘int position’ parameter. The hover explanation was telling that code shouldn’t rely on this position to no vary during execution of the onBindViewHolder execution. It suggested to use the position value immediately and use getAbsoluteAdapterPosition/getBindingAdapterPosition methods afterward.

[the remark this answers is not in the document]
Comment (kekkyojin) [reply app/src/main/java/com/odysee/app/adapter/ClaimListAdapter.java:495]: Can this be merged,@akinwale?

[the remark this answers is not in the document]
Comment (akinwale) [reply app/src/main/java/com/odysee/app/adapter/ClaimListAdapter.java:495]: Sure thing.

**Pull request merged**